src/App.tsx

```diff
@@ -15,47 +16,120 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  NONE,
+  ALPHABET,
+  LENGTH,
+}
+
+type ReorderOptions = {
+  sortType: SortType,
+  isReversed: boolean,
+};
+
+const getReorderedGoods = (
+  goods: string[],
+  { sortType, isReversed }: ReorderOptions,
+) => {
+  const visibleGoods = [...goods];
+
+  if (sortType === SortType.ALPHABET) {
+    visibleGoods.sort((a, b) => a.localeCompare(b));
```

Comment:
what is `a` & `b`?

Reply:
changed a and b to the appropriate names